db_stress_tool/db_stress_gflags.cc

```diff
@@ -547,4 +547,13 @@ DEFINE_bool(use_full_merge_v1, false,
 DEFINE_int32(sync_wal_one_in, 0,
              "If non-zero, then SyncWAL() will be called once for every N ops "
              "on average. 0 indicates that calls to SyncWAL() are disabled.");
+
+DEFINE_int32(verify_db_one_in, 0,
```

Comment:
Should the default to be on and a very large number?

Reply:
Not necessary for `verify_db_one_in`. Probably set `continuous_verification_interval` to 1000 (1 second)?

Reply:
Done